.reflame.config.jsonc

```diff
@@ -306,10 +294,10 @@
         },
       },
       "nodejsCompatibility": {
-        "omitModuleExtension": true,
-        "mapIndexToDirectory": true,
         "packageJson": {
-          "dependenciesSynced": true,
+          "exportsDetection": {
+            "ignoreTopLevelDirectory": true
```

Comment:
This is a bit of a hack to map the build artifact paths (`./dist/*`, which don't exist in our source code) in our package.json exports to source paths relative to `sourceDirectory`.